samples/sqlservices/src/featureProviders/objectExplorerProvider.ts

```diff
@@ -60,13 +63,21 @@ export class ObjectExplorerProvider implements azdata.ObjectExplorerProvider {
 				nodes: [
 					{
 						nodePath: 'root/1',
-						nodeType: 'Database',
-						label: 'abc1',
+						nodeType: '',
```

Comment:
the definition of NodeInfo class needs to be reviewed, it should not be required now, I'll clean it up and add comments to azdata.d.ts later while working with the partner team. this PR is only for adding custom icon support.